docs/app/pages/components/components-sections/date-time-picker/date-range-picker/date-range-picker.component.html

```diff
@@ -172,6 +172,12 @@
     <tr uxd-api-property name="endPickerAriaLabel" type="string" defaultValue="Selecting the end date">
         Defines the aria-live announcement when a date in the end picker is focused.
     </tr>
+    <tr uxd-api-property name="dateLocalization" type="string | Function">
```

Comment:
Could be be more specific with the `Function` type, so instead do: `(date: Date) => string` or whatever it should be